src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -278,6 +279,12 @@ class CollapsibleCourseRun extends React.Component {
     const productSource = courseInfo?.data?.product_source?.slug;
     const courseType = courseInfo?.data?.course_type;
 
+    if (IN_REVIEW_STATUS.includes(courseRun.status) !== isCourseRunInReview) {
+      setTimeout(() => {
+        this.setState({ isCourseRunInReview: IN_REVIEW_STATUS.includes(courseRun.status) });
+      }, 100);
```

Comment:
this 0.1 value -- what was the reason for selecting this value?

Reply:
I tried random values for the timeout, `i.e.` 500, 200, 100, 50. It's minimum value at which this error no longer appears.

Reply:
That is from your local testing, I would suggest keeping the deployment environment in mind when deciding this.